Rewrite `Reporting bad behaviour` (#6051)
ppy/osu-wiki: wiki/Reporting_Bad_Behaviour/en.md

# Reporting bad behaviour

osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) that adheres to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time—do so only as the last resort.

> Accidental reports are ignored. Users who purposedly send invalid reports are [silenced](/wiki/Silence).

Reviewer: ```suggestion
Accidental reports are ignored. Users who deliberately send invalid reports can risk being [silenced](/wiki/Silence).
```

## Special cases

| Nature of infringement | Point of contact |
| :-- | :-- |

> | Mildly offensive private messages | **None**: [ignore the user](/wiki/Chat_Console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Options). |

Reviewer: ```suggestion
| Offensive private messages | **None**: [ignore the user](/wiki/Chat_Console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Options). |
```

| Infringements related to modding and/or mapping | NAT, via the [NAT/BN Management](https://bn.mappersguild.com/reports) website |
| Inappropriate conduct from a [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) member | [Account support team](/wiki/People/The_Team/Account_support_team) at [support@ppy.sh](mailto:support@ppy.sh) |
| Sexual abuse or extreme misbehaviour | Refer to [Reporting Abuse](/wiki/Reporting_Bad_Behaviour/Abuse) |

For anything else, see instructions below.

## In-game chat

### The !report command

![](img/report-command.jpg "Example of using the !report command")

> Chat misbehaviour is the most common infringement. To report it, use the [`!report`](https://osu.ppy.sh/community/forums/topics/34843) command, preferably in the channel where infringement took place. Don't feel stressed: if your message starts with `!report` and a space, **it is only visible to moderators** and no one else in the chat.

Reviewer: ```suggestion
Chat misbehaviour is the most common infringement. To report it, use the [`!report`](https://osu.ppy.sh/community/forums/topics/34843) command, preferably in the channel where infringement took place. Don't feel stressed: if your message starts with `!report` and a space, **it is only visible to moderators** and no one else in the chat. You can also use the report function via the in-game overlay as depicted below if you are still concerned.
```

# Reporting bad behaviour

> osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) who adhere to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time — do so only as the last resort.

Reviewer 2: one team, group or organisation is still singular

```suggestion
osu! has [a team of moderators](/wiki/People/The_Team/Global_Moderation_Team) that adheres to keeping the game and its ecosystem clean and safe. If you notice rule-breaking behaviour or inappropriate content, report it using one of the methods described below. Alternatively, you may contact any of the moderators online directly if your report wasn't properly handled within a reasonable amount of time — do so only as the last resort.
```

Reviewer 2: hmm, just realised that it's ambiguous whether it's the "team" or the "moderators" doing the adhering, depending on where you put the emphasis while reading, so keep it or change it as you prefer.

> Accidental reports are ignored. Users that purposedly send invalid reports are [silenced](/wiki/Silence).

Reviewer 2: ```suggestion
Accidental reports are ignored. Users who purposedly send invalid reports are [silenced](/wiki/Silence).
```

The command's syntax has two forms:

```
!report username
!report username (free-form description follows)
```

If there are spaces in the username, replace them with underscores:

```
!report bad_user spamming in #osu
```

### In-game overlay

![](img/report-user-1.png "User action menu with the report button")

![](img/report-user-2.png "In-game report dialog")

This option works if the offending user is in sight:

1. Click their username and press the red button titled `Report User...`
2. Select the category and add details (optional).

## Website

### Profile page

![](img/report-user-profile.png "The report button on a profile page")

![](img/report-user-web.png "Report dialog on the website")

1. Press the button with three vertical dots and select `Report`.
2. Select the category and add details (optional).

### Forum

![](img/report-user-forum.png "The report button on a forum post")

1. Hover the cursor over the post.

> 2. In a row of buttons that appear in the top right corner of the post, click the button with warning sign.

Reviewer 2: ```suggestion
2. In a row of buttons that appear in the top right corner of the post, click the button with a warning sign.
```
